scipy/signal/filter_design.py

```diff
@@ -161,6 +173,94 @@ def freqs(b, a, worN=None, plot=None):
     return w, h
 
 
+def freqs_zpk(z, p, k, worN=None, plot=None):
```

Comment:
I mean that `freqs` and `freqz` should only exist for backwards/matlab compatibility and the New Way should be `freqresp` and `dfreqresp`.